backend/posts/posts.py

```diff
@@ -1,23 +1,42 @@
 """Functions related to post management and database communcation."""
 
 from typing import List
-from firebase_admin import firestore, initialize_app
+
+import firebase_admin   
+from firebase_admin import firestore, initialize_app, credentials, storage
+import pyrebase
+
+from flask import Flask, render_template, request, flash, redirect, url_for
+from werkzeug.utils import secure_filename
+
 from google.cloud.exceptions import NotFound
 from intelligence import generate_image_metadata
 from models import AddPostMetadata, PostMetadata
 from exceptions import CreatePostException, GetPostException, UnknownPostIdException, DeletePostException
 
 COLLECTION_POSTS = u'/posts'
 
+
+firebaseConfig = {
```

Comment:
Because we're deploying this on Google Cloud Functions, Application Default Credentials will be the means by which we authenticate our access to Firebase APIs.

It's technically okay to check these into version control as they are meant for the web, which is inherently public, but we'll remove these later.